zmon_slr/client.py

```diff
@@ -15,7 +14,12 @@ def __init__(self, url: str, token: str):
         self.token = token
 
         self.session = requests.Session()
-        self.session.headers.update({'Authorization': 'Bearer {}'.format(token), 'User-Agent': 'slr-cli/0.1-alpha'})
+        self.session.headers.update(
+            {
+                'Authorization': 'Bearer {}'.format(token),
+                'User-Agent': 'slr-cli/0.1-alpha',
```

Comment:
is there a version tag we can put here?

Reply:
It was there before. From my part was just automatic formatting change. I believe it is out of scope of this change.